cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/timeseries/FactCodec.java

```diff
@@ -66,11 +67,12 @@ public FactCodec(EntityTable entityTable, int resolution, int rollTimebaseInterv
    * @param tagValues tags
    * @param measureName measure name
    * @param ts timestamp
+   * @param anyAggGroup specifies if we should match any for aggregate group
```

Comment:
"if true, then scan matches every aggregation group; if false, scan matches only aggregation group defined with list of tag values"